homeassistant/components/zwave_me/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "domain": "zwave_me",
+  "name": "Z-Wave.Me",
+  "documentation": "https://github.com/Z-Wave-Me/ha-core",
```

Comment:
Please target the to be created user docs page at home-assistant.io.

Reply:
How and where to create it? Are there examples?

Reply:
Please create a new integration page at https://github.com/home-assistant/home-assistant.io by making a PR to that repo.

Create page docs:
https://developers.home-assistant.io/docs/documenting/create-page

Make sure to base off and target `next` branch in the repo. This is the branch that will go out in the next release. The `current` branch is the current live docs.

Here's an example:
https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/wled.markdown